components/brave_today/common/features.cc

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_today/common/features.h"
+
+#include "base/feature_list.h"
+
+namespace brave_today {
+namespace features {
+
+const base::Feature kBraveNewsFeature {
+  "BraveNews",
+
+#if (!defined(OS_ANDROID) || defined(OS_ANDROID) && defined(OFFICIAL_BUILD))
+      base::FEATURE_ENABLED_BY_DEFAULT
+#else
+      base::FEATURE_DISABLED_BY_DEFAULT
+#endif
```

Comment:
Seems strange that feature is only disabled by default for non official Android builds?

Reply:
the logic would be like this: https://github.com/brave/brave-core/pull/11121#discussion_r750911302
do you think that's ok?

Reply:
sure, as Pete would know requirements here more than me. Thanks

Reply:
I think this is the wrong way around. We want it to be disabled by default for official android builds so we can use griffin to re-enable it for nightly channel only.

```gn
#if (!defined(OS_ANDROID) || (defined(OS_ANDROID) && !defined(OFFICIAL_BUILD)))
      base::FEATURE_ENABLED_BY_DEFAULT
#else
      base::FEATURE_DISABLED_BY_DEFAULT
#endif
```